src/main/java/org/manifold/compiler/front/LiteralExpression.java

```diff
@@ -17,6 +17,10 @@ public TypeValue getType(Scope scope) {
 
   @Override
   public Value getValue(Scope scope) {
+    return getValue();
+  }
+
+  public Value getValue() {

```

Comment:
Not quite, there's one case where we need to call this without a scope (and in fact, pretty soon I'm going to be pushing to remove the `Scope` arguments from these methods).